uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHService.java

```diff
@@ -184,6 +306,14 @@ SshServer getSshServer() {
         return sshd;
     }
 
+    public List<String> getCipherFactoriesNames(){
+        return sshd.getCipherFactoriesNames();
+    }
+
+    public List<String> getMacFactoriesNames(){
```

Comment:
don't see a need for this

Reply:
Is for read the configured Macs, used in the test